MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,276 @@
+
+## Built in validations
+
+The supported built in validations are as follows:
+
+#### Data annotations
+
+You can validate the data using data annotation attributes.
+
+The String type property can be validated using [Required](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=netframework-4.8) and [StringLength](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.stringlengthattribute?view=netframework-4.8) attributes.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Name should not be empty")]
+    [StringLength(15, ErrorMessage = "Name should not exceed 15 characters")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+Refer to this [link] to know more about data annotations in SfDataForm.
+
+#### Date range attribute
+
+You can validate the date time value using date range attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Date)]
+    [DataFormDateRange(MinimumDate = "01/01/2022", MaximumDate = "31/12/2022", ErrorMessage = "Join date is invalid")]
+    public DateTime JoinDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validation mode
+
+The [ValidationMode] determines when the value should be validated.
+
+The supported validation modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm" 
+        ValidationMode="LostFocus">
+    </dataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.ValidationMode = DataFormValidationMode.LostFocus;
+
+{% endhighlight %}
+{% endtabs %}
+
+#### LostFocus
+
+If the validation mode is `LostFocus`, the value will be validated when the editor lost its focus. By default the [ValidationMode] is `LostFocus`
+
+#### PropertyChanged
+
+If the validation mode is `PropertyChanged`, the value will be validated immediately when it is changed.
+
+#### Manual
+
+If the validation mode is `Manual`, the value should be validated manually by calling the [SfDataForm.Validate] or [SfDataForm.Validate(new List<string>())] method.
+
+The following code validates the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.Validate();
+
+{% endhighlight %}
+{% endtabs %}
+
+To validate the specific list of property value, pass the property names as argument.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.Validate(new List<string>() {"FirstName", "Age" });
+
+{% endhighlight %}
+{% endtabs %}
+
+You can determine whether the data form or property is valid or not by using the `Validate` method.
```

Comment:
Determine whether the data form or property is valid by using the `Validate` method.

Reply:
content changed